bin/build.sh

```diff
@@ -13,11 +13,7 @@ create_htaccess() {
     local file="${1}"
     local config="${2}"
 
-    local version
-    version=$(grep version < "${repo_root}/package.json" | \
-        head -1 | awk -F: '{ print $2 }' | sed 's/[",\t ]//g')
-
-    insert_line "# Apache Server Configs v$version | MIT License" "$file"
+    insert_line "# Apache Server Configs | MIT License" "$file"
```

Comment:
Is the version removal intentional here?

Reply:
50% intentional, for now...
I'll try to re-add it later, but without the need for parsing `package.json`. Maybe only for the release distribution also.

Reply:
Actually, yes, it may be better to re-add it now. I'll do this.

Reply:
Since you removed package.json because it's unused, we need another way to track this.

BTW you shouldn't remove package.json IMO. Don't we publish this to npm?

Reply:
Well, I see many people complaining about using npm just for downloading a single file (not specifically for this project, but same), even more for project that do not relate to javascript/Node.js ecosystem.

What do you think about using GitHub' release assets for distribution?

It was relevant to have a `package.json` as the project required some npm dependencies for testing, with this PR this no more the case.

If we still need to publish version to npm, we can add a small package for downloading the file from the release assets (as many other vendors do for this static file distribution situation).

Right? 🙂

Reply:
I don't agree. If we publish to npm, it's the right way to do it. Without any deps or anything, a simple package.json.

Reply:
Alright! 😉 Let me re-add a `package.json`.

Reply:
Let me add it and rebase the branch also

Reply:
It seems you have removed the dist/.htaccess file too. Is that intentional? If so it should be added in .gitignore otherwise reverted.

Reply:
> it should be added in .gitignore

Good catch, added.

Reply:
Thanks for the `package.json`!

Reply:
So, how are the npm deployments going to be made? Because now one will have to build this thing in order to do `npm publish`. I can't being I'm on Windows for example.

Reply:
Ideally with CI automation (or using GitHub Actions 😇). The `.htaccess` is still being deployed to the release on GitHub.

Do you prefer I extract these changes to open another PR? I have to admit, it's not really related to the tests themselves.

Reply:
I don't have any preference as long as npm publishing works.

Reply:
For sure. I added npm as a deploy provider when Travis-CI builds on tags (f1943a5). Let me know if it's OK for you.

(Fyi, I used to build with WSL for previous npm publishing. 😉)